From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Wed, 12 Jun 2019 20:10:31 +0200
Subject: Documentation: remove reference to '_static' folder

The source isn't containing a folder docs/_static so the build of the
documentation is failing otherwise if this config option isn't cleared
out.
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 1195d1f..07eb8d8 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -127,7 +127,7 @@ if on_rtd:
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = []
 
 # If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
 # using the given strftime format.
